src/ng/cacheFactory.js

```diff
@@ -372,7 +372,7 @@ function $CacheFactoryProvider() {
  * the document, but it must be a descendent of the {@link ng.$rootElement $rootElement} (IE,
  * element with ng-app attribute), otherwise the template will be ignored.
  *
- * Adding via the $templateCache service:
+ * Adding via the ```$templateCache``` service:
```

Comment:
I didn't know that it's possible to use single backticks to format as code in markdown. :smile: